contents/blog/posthog-alternatives.md

```diff
@@ -0,0 +1,727 @@
+    	<tr>
+        	<td>A/B testing</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">CRO and test in production</td>
+    	</tr>
+    	<tr>
+        	<td>Group analytics</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Analyze groups (e.g. companies and teams)</td>
+    	</tr>
+    	<tr>
+        	<td>SQL access</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Write your own queries and filters</td>
+    	</tr>
+    	<tr>
+       	<td>Transparent pricing</td>
+      		<td className="text-center"><span className="text-red text-lg">✖</span></td>
+       	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Know exactly what you'll pay</td>
+    	</tr>
+    	<tr>
+       	<td>Billing limits</td>
+      		<td className="text-center"><span className="text-red text-lg">✖</span></td>
+       	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Prevent surprise bills</td>
+    	</tr>
+    	<tr>
+        	<td>Open source</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Open core, build your own integrations</td>
+   		</tr>
+    	<tr>
+        	<td>Cute hedgehogs</td>
+      		<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Joy</td>
+   		</tr>
+	</tbody>
+</table>
+</div>
+
+### Key differences
+
+1. **Breadth of features:** PostHog combines product analytics, session replay, feature management, and A/B testing into one integrated platform. Amplitude is primarily a product analytics tool, and doesn't include any session replay capability.
+
+2. **Marketing use cases:** Amplitude offers more useful features for marketing teams, such as predictive forecasting, and multi-touch attribution. PostHog is primarily designed for technical users, and product teams.
+
+3. **Free tier:** PostHog offers a generous 1 million events and 15k session recordings for free. Amplitude's free plan is limited to 100k monthly tracks users and doesn't include important features, such as cohorts or data tables.
+
+**Read more:** [In-depth PostHog vs Amplitude comparison](blog/posthog-vs-amplitude) 
+
+### What people like
```

Comment:
```suggestion
### What people like about Amplitude
```

Reply:
Yeah, agreed on adding PostHog one (that was in the plan already).

Think I'll just reduce the number of examples rather than summarizing too heavily. I think it's important it's coming from real user comments, rather than us paraphrasing them too much.